response_operations_ui/views/respondents.py

```diff
@@ -50,4 +50,91 @@ def respondent_details(respondent_id):
     ]
 
     respondent['status'] = respondent['status'].title()
-    return render_template('respondent.html', respondent=respondent, enrolments=enrolments, breadcrumbs=breadcrumbs)
+
+    info_message = None
+
+    info = request.args.get('info')
+    if info:
+        info_message = info
+
+    enrolment_changed = request.args.get('info')
+    if enrolment_changed:
+        info_message = info
+    if request.args.get('enrolment_changed'):
+        info_message = 'Enrolment status changed'
+    if request.args.get('account_status_changed'):
+        info_message = 'Account status changed'
+
+    return render_template('respondent.html', respondent=respondent, enrolments=enrolments, breadcrumbs=breadcrumbs,
+                           info_message=info_message)
+
+
+@respondent_bp.route('/edit-contact-details/<respondent_id>', methods=['GET'])
+@login_required
+def view_contact_details(respondent_id):
+    respondents_details = party_controller.get_respondent_by_party_id(respondent_id)
+
+    form = EditContactDetailsForm(form=request.form, default_values=respondents_details)
+
+    return render_template('edit-contact-details.html', respondent_details=respondents_details,
+                           form=form, tab='respondents')
+
+
+@respondent_bp.route('/edit-contact-details/<respondent_id>', methods=['POST'])
+@login_required
+def edit_contact_details(respondent_id):
+    form = request.form
+    contact_details_changed = party_controller.update_contact_details(respondent_id, form)
+
+    ui_message = 'No updates were necessary'
```

Comment:
Same as above. Flash might be better here.
Additionally, I think this is the wrong way to do it.  This kind of code is just asking for weird bugs to occur when multiple things are provided and the message keeps changing.

Reply:
added now